aws/aws_instance/ami_lookup/main.tf

```diff
@@ -0,0 +1,54 @@
+# Summary: Data Source for getting latest getting AMI Linux Imagehttps://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html
+
+# Documentation: https://www.tserraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 1.0.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.38"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+provider "aws" {
+  region = "us-east-1"
+  default_tags {
+    tags = {
+      cs_terraform_examples = "aws_instance/ami_lookup"
+    }
+  }
+}
+
+# Documentation: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami
+# Explanation: Get most_recent version of Ubuntu Image 
+data "aws_ami" "changeme_aws_ami" {
+  most_recent = true
+
+  # Explanation: Canonical now publishes  official Ubuntu images on the Amazon cloud. Is it possible to get parameters with AWS SSM (System Manager)
```

Comment:
`Is it possible to get parameters with AWS SSM (System Manager)` not sure what this means, can you elaborate.

Is it needed? If not, can it be removed?

Reply:
Sentence was removed